test/verify/check-metrics

```diff
@@ -14,43 +14,43 @@ from testlib import *
 from machine_core.constants import TEST_OS_DEFAULT
 
 
-def getMaximumSpike(browser, g_type, saturation, hour, minute):
+def getMaximumSpike(self, g_type, saturation, hour, minute):
```

Comment:
This is weird for a non-method. Better call this `test` or so.

Reply:
You meant like this? (pushed)

Reply:
Right, thanks!